troubleshooting/js-errors.md

```diff
@@ -51,6 +51,19 @@ You can check if this is the case by inspecting the Network tab of your browser
 
 * A problem occurs during the deployment. See the [Deployment Troubleshooting]({%slug deployment-troubleshooting%}) article for more details.
 
+* The application is navigating from an interactive page. If this is the case, register the `telerik-blazor.js` after the `<base href="/" />` in the `<head>` of the web page:
+
+<div class="skip-repl"></div>
+
+````HTML
```

Comment:
Double-check the opening and closing syntax here. Usually it's three \` symbols.